doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -76,20 +76,20 @@ are only a handful of key differences between the two.
    -  For ``matrix``, **'``*``\ ' means matrix multiplication**, and the
       ``multiply()`` function is used for element-wise multiplication.
 
--  Handling of vectors (rank-1 arrays)
+-  Handling of vectors (one-dimensional arrays)
 
-   -  For ``array``, the **vector shapes 1xN, Nx1, and N are all
-      different things**. Operations like ``A[:,1]`` return a rank-1
-      array of shape N, not a rank-2 of shape Nx1. Transpose on a rank-1
-      ``array`` does nothing.
-   -  For ``matrix``, **rank-1 arrays are always upconverted to 1xN or
-      Nx1 matrices** (row or column vectors). ``A[:,1]`` returns a
-      rank-2 matrix of shape Nx1.
+   -  For ``array``, the **vector shapes 1xN, Nx1, and N are all different
+      things**. Operations like ``A[:,1]`` return a one-dimensional array of
+      shape N, not a two-dimensional array of shape Nx1. Transpose on a
+      one-dimensional ``array`` does nothing.
+   -  For ``matrix``, **one-dimensional arrays are always upconverted to 1xN
+      or Nx1 matrices** (row or column vectors). ``A[:,1]`` returns a
+      two-dimensional matrix of shape Nx1.
 
--  Handling of higher-rank arrays (rank > 2)
+-  Handling of higher-dimensional arrays (ndim > 2)
 
-   -  ``array`` objects **can have rank > 2**.
-   -  ``matrix`` objects **always have exactly rank 2**.
+   -  ``array`` objects **can have number of dimensions > 2**;
```

Comment:
Personally I like `;` for ending list elements, but I guess you're asking why here and not the same edit elsewhere?